scripts/ci/iptables-restore

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]; do
+	key="$1"
+	case "$key" in
+		-w?*)
+			shift
```

Comment:
you may want to fix the problem, not ignore it
                POSITIONAL+=("-w")
                POSITIONAL+=("${key#-w}")

Reply:
ok